action.yml

```diff
@@ -71,10 +55,9 @@ runs:
       if: steps.check-update.outputs.files_changed == 'true'
       id: build-lean
       continue-on-error: true
-      run: |
-        lake exe cache get || true
-        lake build
-      shell: bash
+      uses: Seasawher/lean-action@patch-1
```

Comment:
The current implementation of lean-action was problematic and needed to be corrected. Once the fixes are merged, I will use leanprover/lean-action.